lib/opensusebasetest.pm

```diff
@@ -213,9 +213,11 @@ sub wait_boot {
 
     # Reset the consoles after the reboot: there is no user logged in anywhere
     reset_consoles;
-    assert_screen "bootloader", $bootloader_time;
+    if (get_var("OFW")) {
+        assert_screen 'bootloader', $bootloader_time;
```

Comment:
If you want to remove bootloader-ofw then remove it everywhere not only on one place